cmd/livepeer/starter/starter.go

```diff
@@ -1412,3 +1420,37 @@ func getBroadcasterPrices(broadcasterPrices string) []BroadcasterPrice {
 
 	return pricesSet.Prices
 }
+
+type KeystorePath struct {
+	path    string
+	address ethcommon.Address
+}
+
+func ParseEthKeystorePath(ethKeystorePath string) (KeystorePath, error) {
```

Comment:
```suggestion
func parseEthKeystorePath(ethKeystorePath string) (KeystorePath, error) {
```

I think we don't need to have it accessible outside this package.